src/app/modules/webhook/webhook.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { NextFunction, Request, Response } from 'express'
+
+import { pushData } from './webhook.service'
+import { WebhookRequestLocals } from './webhook.types'
+
+/**
+ * POST submission to a specified URL.  Only works for encrypted submissions.
+ * The webhook is fired on a best-effort basis, so the next middleware
+ * is always called.
+ * @param {Express.Request} req Express request object
+ * @param {Object} req.form The form object containing the webhook URL
+ * @param {Object} req.submission The submission saved to the database
+ * @param {Express.Response} res Express response object
+ * @param {function} next Next middleware
+ */
+export const post = (
+  req: Request & WebhookRequestLocals,
+  res: Response,
+  next: NextFunction,
+) => {
+  // TODO: Once we move away from the middleware pattern, there should not be a webhook controller
```

Comment:
Create an issue for this and tag it please, try to move away from dangling TODOs

Reply:
agreed good point

Reply:
https://github.com/opengovsg/FormSG/issues/149